Landing CV/index.html

```diff
@@ -0,0 +1,305 @@
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                    </div>
+                </div>
+            </div>
+            <div class="skills">
+                <img src="img/SASS.svg" alt="SASS">
+                <div class="skills_info">
+                    <span>SASS</span>
+                    <div class="stars">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/Star_gray.svg" alt="star_gray">
+                    </div>
+                </div>
+            </div>
+            <div class="skills">
+                <img src="img/git 1.svg" alt="git_1">
+                <div class="skills_info">
+                    <span>Git</span>
+                    <div class="stars">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/Star_gray.svg" alt="star_gray">
+                        <img src="img/Star_gray.svg" alt="star_gray">
+                    </div>
+                </div>
+            </div>
+            <div class="skills">
+                <img src="img/javascript 1.svg" alt="JS">
+                <div class="skills_info">
+                    <span>JavaScript</span>
+                    <div class="stars">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/Star_gray.svg" alt="star_gray">
+                        <img src="img/Star_gray.svg" alt="star_gray">
+                    </div>
+                </div>
+            </div>
+            <div class="skills">
+                <img src="img/github-1 1.svg" alt="github">
+                <div class="skills_info">
+                    <span>Github</span>
+                    <div class="stars">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/star.svg" alt="star">
+                        <img src="img/Star_gray.svg" alt="star_gray">
+                    </div>
+                </div>
+            </div>
+        </div>
+                <!----------Skills_hide---------->
+        <div class="skills_panel_hide">
+            <input type="checkbox" class="read-more-checker" id="read-more-checker" />
```

Comment:
_Can't make a normal arrow. What can you read to make it clearer?_
Use the same checkbox after all skills (open and hidden). Depending on its state, use rotation (transform: rotate (180deg);